components/BottomMenu.js

```diff
@@ -88,7 +66,7 @@ function BottomMenu(props) {
 
     const goToMoreDetails = () => {
         props.navigation.navigate("MoreDetails", {
-            name: selectedBuilding
+            name: store.getState().selectedBuildingName
```

Comment:
> > I found a small bug. When searching Men's/Women's Washroom or Water Fountain (Indoor POI), the app responds with:
> > `null is not an object (evaluating 'searchItem.coordinates.latitude')`
> 
> It could be added to the DoubleSearch search bars manually instead, just like we did with current location and point of interest, and removed as soon as we pressed on the back button.

I fixed it differently, you would be able to see it in one of my latest commits

Reply:
It's a feature of redux, I initialized one at redux/reducers/index.js. I could explain further in pm.